src/main/java/frc/robot/util/ShuffleBoard/ShuffleBoardManager.java

```diff
@@ -25,7 +25,7 @@ public class ShuffleBoardManager {
     public ShuffleBoardManager(Drivetrain drive){
         driveTab = new DriveTab(drive);
         autoTab = new AutoTab(drive);
-        subsystemTab = new SubsystemTab();
+        subsystemTab = new SubsystemTab(0);
```

Comment:
You need to add a new parameter to the ShuffleboardManager constructor for your Ting and then pass it into subsystemTab.